src/trie.c

```diff
@@ -107,7 +135,6 @@ trie_t *trie_get_subtrie(trie_t *t, char* word)
     return curr;
 }
 
-
 int trie_search(trie_t *t, char* word)
```

Comment:
Let's rename this to `trie_contains` and have it return a `bool`. Note that this will require changing all callers: you can use `grep -r trie_search .` to find all occurrences of `trie_search`.